src/plugins/cosmos/CosmosAssetAccountDetails.tsx

```diff
@@ -22,6 +22,29 @@ type AssetDetailsProps = {
 
 export const CosmosAssetAccountDetails = ({ assetId: caip19, accountId }: AssetDetailsProps) => {
   const cosmosInvestorFlag = useAppSelector(state => selectFeatureFlag(state, 'CosmosInvestor'))
+
+  // TODO: wire up with real validator data
+  const stakingOpportunities = [
+    { id: 1, validatorName: 'Cosmos Validator', apr: 0.12 },
+    {
+      id: 2,
+      validatorName: 'Cosmos Validator',
```

Comment:
That'll probably be named `moniker` unless we do some transformations on the to-be-made query function/selectors: https://github.com/shapeshift/lib/pull/491/files#diff-e72c79aae6dd3ad84fe702284223d9f1ecee635105eec56322a54718fbc7310fR13